src/modules/85-cv/components/FormikCheckboxGroup/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { FieldArray } from 'formik'
+import { Checkbox, FormError } from '@harness/uicore'
+import type { IOptionProps } from '@blueprintjs/core'
+
+interface FormikCheckboxGroupProps {
+  name: string
+  items: IOptionProps[]
+  values: Array<string | number>
+  errorMessage?: string
+}
+
+export default function FormikCheckboxGroup({
+  name,
+  items,
+  values,
+  errorMessage
+}: FormikCheckboxGroupProps): JSX.Element {
+  return (
+    <>
+      <FieldArray
+        name={name}
+        render={arrayHelpers =>
+          items.map(tag => (
+            <Checkbox
+              name="checkboxGroup"
+              value={tag.value}
+              label={tag.label}
+              key={tag.value}
+              checked={values?.includes(tag.value)}
+              onChange={e => {
+                if (e.currentTarget.checked) {
+                  arrayHelpers.push(tag.value)
+                } else {
+                  const idx = values?.indexOf(tag.value)
+                  arrayHelpers.remove(idx)
```

Comment:
should we be calling remove if idx is -1 ?
```suggestion
                values?.indexOf(tag.value) &&  arrayHelpers.remove(values?.indexOf(tag.value))
```

Reply:
deprecated